cicd/gitlab/parts/dockerbuild.genomio.gitlab-ci.yml

```diff
@@ -0,0 +1,122 @@
+# See the NOTICE file distributed with this work for additional information
+# regarding copyright ownership.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Python related part of the Ensembl/ensembl-genomio CI/CD pipeline
+
+# Test stage instances
+
+# A generic job to be used by all Python jobs with the "extends" mechanism
+# (https://docs.gitlab.com/ee/ci/yaml/#extends)
+
+### variable setup
+variables:
+  # https://docs.gitlab.com/ee/user/namespace/  
+  CONTAINER_IMAGE_NAME: ensembl-genomio
+  GENOMIO_DIR: containers/docker/genomio
+  GENOMIO_RELEASE_VERSION: "v1.3.1"
```

Comment:
Can this be grabbed by the tag name by any chance? Whilst releases are not mirrored by GitLab, tags are.